leakcanary-android-core/src/main/java/leakcanary/internal/DisplayLeakConnectorView.kt

```diff
@@ -80,18 +81,18 @@ internal class DisplayLeakConnectorView(
         .toFloat()
 
     classNamePaint = Paint(Paint.ANTI_ALIAS_FLAG)
-    classNamePaint.color = resources.getColor(R.color.leak_canary_class_name)
+    classNamePaint.color = ContextCompat.getColor(context, R.color.leak_canary_class_name)
     classNamePaint.strokeWidth = strokeSize
 
     leakGroupRootPaint = Paint(Paint.ANTI_ALIAS_FLAG)
-    leakGroupRootPaint.color = resources.getColor(R.color.leak_canary_class_name)
+    leakGroupRootPaint.color = ContextCompat.getColor(context, R.color.leak_canary_class_name)
```

Comment:
Is there any advantage to this specific code? If something is deprecated but right, then the people who deprecated it are wrong :)

Reply:
Agreed; for me it's worth it to resolve the warning but otherwise 🤷‍♂